jest.conf/visual.testUtils.js

```diff
@@ -0,0 +1,43 @@
+import percySnapshot from '@percy/puppeteer';
+
+export async function renderComponent(component, props) {
+  const initialState = await page.evaluate(() => {
+    const testing_playground = document.querySelector('#testing-playground');
+    return testing_playground ? testing_playground.innerHTML : '';
+  });
+  await page.evaluate(
+    ({ component, props }) => {
+      window.postMessage(
+        {
+          type: 'RENDER_COMPONENT',
+          component: component,
+          props: props,
+        },
+        '*'
+      );
+    },
+    { component, props }
+  );
+  await page.waitForSelector('#testing-playground');
+
+  await page.waitForFunction(
```

Comment:
It would be nice if we add here some comments of the motivation for this check, so anybody else who read this code knows the intention of this wait.

Reply:
Sure. I'll add the comments :+1: